packages/packages/documents/src/__tests__/test-utils.ts

```diff
@@ -1,15 +1,30 @@
-import { Document } from '../';
+import { Document } from '../types';
 
-export function createMockDocument(): Document {
+export function createMockDocument( {
+	id = 1,
+	title = `Document ${ id }`,
+	status,
+	type,
+	isDirty = false,
+	isSaving = false,
+	isSavingDraft = false,
+	userCan,
+}: Partial<Document> = {} ): Document {
 	return {
-		id: 1,
-		title: 'Document 1',
-		status: 'publish',
-		type: 'wp-page',
-		isDirty: false,
-		isSaving: false,
-		isSavingDraft: false,
-		userCan: {
+		id,
+		title,
+		status: status || {
+			value: 'publish',
+			label: 'Published',
+		},
+		type: type || {
+			value: 'wp-page',
+			label: 'Page',
+		},
+		isDirty,
+		isSaving,
+		isSavingDraft,
+		userCan: userCan || {
```

Comment:
Maybe move the default with all the reset at the function arguments?

Reply:
Done